src/backend/marsha/core/api.py

```diff
@@ -406,25 +407,46 @@ def post(self, request):
 
         consumer_site = video.playlist.consumer_site
 
+        # We want to allow to put xapi statement in a logger. We have a here a convention name
+        # specific allowing a have a logger by consumer site. If you want to log xapi statements
+        # you have to prefix the consumer site domain with `xapi.`. For example for the consumer
+        # site with domaine name foo.education you have to create a logger name
+        # `xapi.foo.education` and configure as you want the associated handler and formatter
+        # depending where you want to log the statements.
+        xapi_logger = logging.getLogger(f"xapi.{consumer_site.domain}")
+
+        # xapi statement sent by the client but incomplete
+        partial_xapi_statement = serializers.XAPIStatementSerializer(data=request.data)
+
+        if not partial_xapi_statement.is_valid():
+            return Response(partial_xapi_statement.errors, status=400)
+
+        try:
+            # xapi statement enriched with video and lti_user informations
+            xapi_statement = XAPIStatement(
+                video, partial_xapi_statement.validated_data, lti_user
+            )
+        # pylint: disable=invalid-name
```

Comment:
I'm curious about the Error name invalidity.

Reply:
With recent version of pylint there is no more error...